ase/rtl/platform.vh

```diff
@@ -124,34 +124,32 @@
 `define LAT_UNDEFINED              300
 
 `define RDWR_VL_LATRANGE           20,118
-`define RDWR_VH_LATRANGE           240,270
+`define RDWR_VH_LATRANGE           140,180
```

Comment:
how are these values chosen?

Reply:
A balance of accuracy vs. simulator speed. The right latency relative to the size of the simulated pipeline gives a reasonable number of requests in flight.